types/office-js/index.d.ts

```diff
@@ -551,6 +551,37 @@ declare namespace Office {
         */
         asyncContext: any;
         /**
+        * Gets an object that may provide additional information if an error occurred.
```

Comment:
"Gets an object" -> "An object"
(Unless this is generated when called, but then this should be a method and not a property)